src/types/name.rs

```diff
@@ -34,7 +92,7 @@ pub enum NameAttribute<'a> {
     Custom(Cow<'a, str>),
 }
 
-impl NameAttribute<'static> {
+impl<'a> NameAttribute<'a> {
```

Comment:
Why was this changed? This should be able to still return `<'static>`.

Reply:
Oops. :)